src/renderer/components/PerCurrencySelectAccount/state.js

```diff
@@ -46,6 +46,14 @@ const getIdsFromTuple = (accountTuple: AccountTuple) => ({
   subAccountId: accountTuple.subAccount ? accountTuple.subAccount.id : null,
 });
 
+export type useCurrencyAccountSelectReturnType = {
+  availableAccounts: Array<AccountTuple>,
+  currency: ?CryptoCurrency | TokenCurrency,
+  account: ?Account | any,
```

Comment:
I prefer we use the account / parentAccount naming convention since we use it throughout the whole codebase.
And `SubAccount` might give you problems since it will try to allow for `ChildAccount` too, that's why we were using `TokenAccount | Account` elsewhere, if you can make it work like that it's fine by me, but account/parentAccount 🙏🏼

Reply:
Oh right, I'm going to rework that hook and replace `SubAccount` with `parentAccount`. Hope other pieces of code that use it (outside of the swap) is covered by tests 👀

Reply:
Done!